mipengine/node/monetdb_interface/common_actions.py

```diff
@@ -258,7 +213,7 @@ def _convert_mip2monetdb_column_type(column_type: str) -> str:
     return type_mapping.get(column_type)
 
 
-def _convert_monetdb2mip_column_type(column_type: str) -> str:
+def _convert_monet2mip_column_type(column_type: str) -> str:
```

Comment:
This conversion refers to mapping monetdb types to python types, right? It would be ideal to not have any refference to MIP inside the code where is not needed. Is it really needed here(or in any other place) to use the word MIP?

Reply:
Yes, the correct name should involve python, not the MIP. But since we use these conversions in various places, we should consider using [`pymonetdb.sql.monetize` and `pymonetdb.sql.pythonize`](https://github.com/gijzelaerr/pymonetdb/tree/master/pymonetdb/sql).

Reply:
The "MIP" types are the variable types used in the MIP metadata, in the Data Catalog, in the MIP frontend and maybe other places. 

These are not types used only inside the MIP-Engine.

Conceptually they are not linked to the MIP-Engine and it's coding language. Before we were coding in Java and were using the same types. Maybe now with python we can have an exact relation of "MIP" types to python types but it should be clear that these are not just python types.

I think it's more clear to have a MIP type converter since it makes it easier to change it in the future if needed. Inside the method we can use `pymonetdb.sql.monetize` and `pymonetdb.sql.pythonize` for now if the mapping is exact.

Reply:
I think `bool` and `clob` were added from @KFilippopolitis at the beginning to try and have all the possible types.

As you said in the MIP we only have `real`, `int` and `text` so the rest can be removed.

About the python types, I am not sure I understand the connection. These types are for the data that will "live" inside monet. We use the "MIP" types because they are provided from the metadata/frontend with those enumerations and then we convert them to "monet" types so we can use the actual data in the database.

We never actually use these data in python so I am not sure why we need a link between monet types and python types or mip types and python types.

Reply:
Also, what these methods aim to do is exactly what you suggest. 

> In any case a unified set of methods for converting among MIP, Python and MonetDB types is needed.

We can see if a conversion to python is needed and add that as well.

Reply:
Thanks for reminding "bool" and "clob" was added for a specific function that we no longer use so now they will be removed.